powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/repository/SignatureAuditRepository.java

```diff
@@ -38,7 +38,7 @@
      * @param endingDate   Ending date (date "to").
      * @return List of {@link SignatureEntity} instances.
      */
-    List<SignatureEntity> findByActivation_UserIdAndTimestampCreatedBetweenOrderByTimestampCreatedDesc(String userId, Date startingDate, Date endingDate);
+    List<SignatureEntity> findByActivation_UserIdAndTimestampCreatedBetweenOrderByIdDesc(String userId, Date startingDate, Date endingDate);
```

Comment:
IMO this will not work in some Oracle configurations where [Oracle may decide to generate ID sequence out of order](https://stackoverflow.com/questions/13513936/oracle-sequence-value-are-not-ordered). With our current `hibernate_sequence`, this should not happen. However, [we would like to migrate to more lose entity ID generation](https://github.com/lime-company/powerauth-server/issues/7), and then this could cause problem.

I suggest ordering records by timestamp and then by ID (to distinguish order for records in the same second)....

Reply:
Ok, makes sense, I will update the method name.